src/components/NavHeader.vue

```diff
@@ -0,0 +1,27 @@
+<template>
+  <section class="banner">
+    <RouterLink to="/">
+      <img src="../assets/images/white-logo-without-text.png" alt="logo" />
+    </RouterLink>
+    <HamburgerMenu />
+  </section>
+</template>
+
+<script setup>
+import HamburgerMenu from "./HamburgerMenu.vue";
+</script>
+
+<style scoped>
+.banner {
+  height: 10vh;
+  width: 100%;
+  margin-bottom: 2rem;
+  position: relative;
+}
+
+img {
+  width: 12vh;
+  padding-top: 1rem;
+  padding-left: 1rem;
```

Comment:
Here we set the width, but we use viewport-height as relative unit of measurement.

Reply:
Or is this viewport-horizontal?